editor/effects.js

```diff
@@ -19,9 +19,10 @@ export default {
 	REQUEST_POST_UPDATE( action, store ) {
 		const { dispatch } = store;
 		const { postId, edits } = action;
-		const isNew = ! postId;
 		const toSend = postId ? { id: postId, ...edits } : edits;
+		const isNew = ! postId;
 
+		dispatch( { type: 'CLEAR_POST_EDITS' } );
```

Comment:
Hard problem to solve, we should probably diff the edits before and after save to avoid loosing the unsaved edits (if we move this call in the onSuccess)